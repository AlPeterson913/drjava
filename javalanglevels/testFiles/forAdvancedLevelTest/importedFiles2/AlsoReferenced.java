// Language Level Converter line number map: dj*->java. Entries: 4
//     1->3         2->4         3->5         4->6    
package forAdvancedLevelTest.importedFiles2;

public class AlsoReferenced {
}
