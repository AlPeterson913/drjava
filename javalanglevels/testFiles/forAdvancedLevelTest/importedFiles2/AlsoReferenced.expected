package forAdvancedLevelTest.importedFiles2; //[1]
//[2]
public class AlsoReferenced { //[3]
} //[4]